hermes/bin/src/runtime_extensions/hermes/http_gateway/event.rs

```diff
@@ -0,0 +1,73 @@
+//! HTTP-Gateway handler implementation.
+
+use std::sync::mpsc::Sender;
+
+use hyper::{self, body::Bytes};
+use serde::{Deserialize, Serialize};
+
+use crate::event::HermesEventPayload;
+
+/// HTTP response code
+type Code = u16;
+
+/// Headers in kv form
+pub type HeadersKV = Vec<(String, Vec<String>)>;
+
+/// HTTP Path
+type Path = String;
+
+/// HTTP method e.g GET
+type Method = String;
+
+/// Req body
+type Body = Vec<u8>;
+
+#[derive(Serialize, Deserialize, Debug)]
+/// Msg type for MPSC
```

Comment:
I think we should follow a convention that all Rust code should be together and comments not split it.
```suggestion
#[derive(Serialize, Deserialize, Debug)]
```